CONTRIBUTING.md

```diff
@@ -1,11 +1,11 @@
-## Contributing
+## Contributing to the _GitHub Extension for Visual Studio_
```

Comment:
My logic was that I was trying to pull focus on the project title and as the whole title was already bold it really only left me with italics, but I've dropped it out now.